_docs/_latest/configure/config-validator/policy-library-sync-from-git-repo.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Policy Library Sync from Git Repository
+order: 702
+---
+
+# {{ page.title }}
+
+This page describes how to sync policies from Git repository.
+
+---
+
+## **Policy Library Sync from Git Repository**
+
+As part of the Terraform configuration, you will need to include a few 
+additional variables to enable the git-sync feature.
```

Comment:
Add some info to explain that by default, commits to the master branch will get synced, but this can be controlled with the `policy_library_repository_branch` input. Also, include as a note (maybe at the bottom) that after policies are synced over, the CV service will need to be restarted in order to use the policies. This is done automatically by the cron job (run_forseti), but if people are running ad-hoc scans, then they will need to restart CV.

Reply:
Done